docs/reference/requirement-sets/excel-api-1-4-requirement-set.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Excel JavaScript API requirement set 1.44
+description: 'Details about the ExcelApi 1.4 requirement set'
+ms.date: 07/11/2019
+ms.prod: excel
+localization_priority: Normal
+---
+
+# What's new in Excel JavaScript API 1.4
+
+The following are the new additions to the Excel JavaScript APIs in requirement set 1.4.
+
+## Named item add and new properties
+
+New properties:
+
+* `comment`
+* `scope` worksheet or workbook scoped items
+* `worksheet` returns the worksheet on which the named item is scoped to.
+
+New methods:
+
+* `add(name: string, reference: Range or string, comment: string)`Adds a new name to the collection of the given scope.
+* `addFormulaLocal(name: string, formula: string, comment: string)` Adds a new name to the collection of the given scope using the user's locale for the formula.
+
+## Settings API in the Excel namespace
+
+The [Setting](/javascript/api/excel/excel.setting) object represents a key:value pair for a setting persisted to the document. The functionality of `Excel.Setting` is equivalent to `Office.Settings`, but uses the batched API syntax, rather than the Common API's callback model.
+
+APIs include `getItem()` to get setting entry via the key, `add()` to add the specified key:value setting pair to the workbook.
+
+## Others
+
+* Set table column name (prior version only allows reading).
```

Comment:
It is weird to me to say prior version allows because I want to read it as allowed but I guess because it is still available you have used allows here?